config/routes.rb

```diff
@@ -3,6 +3,9 @@
   devise_for :users, controllers: {sessions: "sessions"}
   
   root 'components#index'
+  
+  resources :teams, only: %i[index new] do
```

Comment:
could we move the team paths under a `/admin/` route please?

Reply:
Ok, thanks